app/models/product.rb

```diff
@@ -253,10 +253,16 @@ def product_type
     self.class.name.underscore.pluralize
   end
 
+  # Used when displaying quantities throughout the site and when editing an order.
   def quantity_as_time?
     false
   end
 
+  # Primarily used when adding to an existing order (merge orders)
+  def order_quantity_as_time?
```

Comment:
So this separate method is used to distinguish TimedService behavior from Instrument behavior then?

Reply:
Yes. Just using the existing `quantity_as_time?` was causing instruments to show the duration field.